package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -145,18 +145,13 @@
 
 logger = logging.getLogger("MDAnalysis.coordinates.AMBER")
 
+from scipy.io import netcdf
```

Comment:
`netcdf` and `netCDF4` are very similar variable names, maybe they should be `netcdf as netcdf_scipy` and `netCDF4 as netCDF4_official`?  Just thinking if I ever have to work on this module, that's what I'll want